### PR TITLE
Update DG for feeding times feature

### LINKED ISSUE #105: Update DG after implementing features
Individually, update your portion of the DG after implementing your feature. Last man closes issue.

### PR DESCRIPTION
**Pull request by koonweee**
Resolve #105

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -230,6 +230,75 @@ parses the user's input as a file name and then creates a `SnapCommand` object w
...
+
+The custom comparator `FeedTimeComparator` compares the integer values of the feeding times, returning them in ascending order.
+
+The feeding times feature allows for the following functionallity:
+
+* Add multiple feeding times to each animal listing.
+* Ensure feeding times are always displayed in chronological order.
+
+The following notable methods are used for the feeding times feature:
+* `parseFeedTimes` (in `ParserUtil` class) - returns a Set of `FeedTime` objects from user input
```
Comment (lamlaaaam) [inline docs/DeveloperGuide.md]: For consistency, name methods in form Class#method(parameter). e.g. ParserUtil#parseFeedTimes()

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -230,6 +230,75 @@ parses the user's input as a file name and then creates a `SnapCommand` object w
...
+
+[image of state]
+
+Step 4. The `ArgumentTokenizer` class returns the feeding times found in the users input. A set of `FeedTime` objects is created by the `parseFeedTimes` method in the `ParserUtil` class.
+
+[image of state]
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** A ParseException is thrown by parseFeedTimes if the feeding time input does not match the defined format.
+</div>
+<div markdown="span" class="alert alert-info">:information_source: **Note:** The TreeSet created by parseFeedTimes utilizes the FeedTimeComparator, ensuring that the set is returned in chronoligcal order.
```
Comment (lamlaaaam) [inline docs/DeveloperGuide.md]: Typo in "chronological".

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -230,6 +230,75 @@ parses the user's input as a file name and then creates a `SnapCommand` object w
...
+</div>
+
+Step 5. An `Animal` object is created with the Set of `FeedTime` objects
```
Comment (lamlaaaam) [inline docs/DeveloperGuide.md]: Missing a period at the end.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (koonweee) [reply docs/DeveloperGuide.md]: Updated to match format

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (koonweee) [reply docs/DeveloperGuide.md]: Resolved

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -230,6 +230,75 @@ parses the user's input as a file name and then creates a `SnapCommand` object w
...
+
+The custom comparator `FeedTimeComparator` compares the integer values of the feeding times, returning them in ascending order.
+
+The feeding times feature allows for the following functionallity:
+
+* Add multiple feeding times to each animal listing.
+* Ensure feeding times are always displayed in chronological order.
+
+The following notable methods are used for the feeding times feature:
+* `ParserUtil#parseFeedTimes(feedTimes)` - returns a Set of `FeedTime` objects from user input
```
Comment (lamlaaaam) [inline docs/DeveloperGuide.md]: Parameters should be class names.

**Pull request merged**